apiserver/registration.go

```diff
@@ -205,7 +205,7 @@ func (h *registerUserHandler) getSecretKeyLoginResponsePayload(
 	caCert, _ := controllerConfig.CACert()
 	payload := params.SecretKeyLoginResponsePayload{
 		CACert:         caCert,
-		ControllerUUID: st.ControllerUUID(),
+		ControllerUUID: controllerConfig.ControllerUUID(),
```

Comment:
There is a chain through `httpContext` -> `Server` -> `sharedServerContext`. I think you should be able to get it from there without even needing the service.

Reply:
I saw that, but went with the config since we use it anyway here for the `CACert` and the chain is pretty ugly.

Reply:
I missed that we already had it. This is fine.